### PR TITLE
Add backend healthcheck

### PR DESCRIPTION
**Pull request by mathcolo**
The train tracker issue this morning reminded me that the dashboard needs a real health check. This checks to make sure the API key has a non-zero length, and that it can get alerts from S3. 500 error if a check fails, and a number output indicating which one.

### CODE REVIEW

**Review by nathan-weinberg — APPROVED** [review]

**Comment by austinjpaul** [comment]
This looks decent. Is the goal to point uptime bot at it or something so we get slack notifications? In general, i'd love a log of 500s (and maybe 400s?) that occur- I tried to find such things in aws, but it made me cry. Do you know of any way to get notifications of these? either in slack or a weekly digest email or something?

File: server/app.py [hunk 1]
```diff
@@ -47,6 +47,31 @@ def mutlidict_to_dict(mutlidict):
     return res_dict
 
 
+@app.route("/healthcheck", cors=cors_config)
+def healthcheck():
+    # These functions must return True or False :-)
+    checks = [
+        lambda: len(secrets.MBTA_V2_API_KEY) > 0,
+        lambda: "2020-11-07 10:33:40" in json.dumps(data_funcs.headways(date(year=2020, month=11, day=7), ["70061"]))
```
Comment (austinjpaul) [inline server/app.py:55]: this is an old date, so testing s3 access i presume. Should we test mbta's api as well? and/or alert cache thingys?

**Review by austinjpaul — APPROVED** [review]

**Comment by nathan-weinberg** [comment]
Any way to incorporate this into a CI check? We can always brainstorm this later ofc

File: server/app.py (hunk 1 above)
Comment (mathcolo) [reply server/app.py:55]: Yes we can! I'll merge this and can do a subsequent PR with some additional features

**Comment by mathcolo** [comment]
@nathan-weinberg You bet; let's add more soon

**Pull request merged**